install_config/master_node_configuration.adoc

```diff
@@ -1422,6 +1410,33 @@ start on a machine that does not have docker started.
 
 |===
 
+[[node-config-volume-config]]
+=== Volume Configuration
+
+You can use the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/storage_administration_guide/ch-xfs[XFS quota subsystem] 
+to limit the number of `emptyDir` volumes and volumes based on an `emptyDir` volume, such as secrets and configmaps, on each node.
+
+Enable a quota on local storage for each unique xref../architecture/additional_concepts/authorization.adoc#security-context-constraints[FSGroup ID] 
+using the *_/etc/origin/node/volume-config.yaml_* file.
+
+----
+apiVersion: kubelet.config.openshift.io/v1
+kind: VolumeConfig
+localQuota: <1>
+  perFSGroup: 50 <2>
+---
+
+<1> Contains the options for controlling local volume quota on the node.
+<2> Sets the maximum number of `emptyDir` volumes for each unique FSGroup ID. 
```

Comment:
this is not right.

Specifies the maximum amount of storage that can be used for disk-based `emptyDir` volumes on the node for each unique FSGroup ID